ariadne/error_formatter.py

```diff
@@ -0,0 +1,34 @@
+from traceback import format_tb
+
+SKIP_TRACEBACK_HEAD = 3
+
+
+def format_error(error, debug=False):
+    formatted = error.formatted
+    if debug:
+        if "extensions" not in formatted:
+            formatted["extensions"] = {}
+        formatted["extensions"]["exception"] = get_error_details(error)
+    return formatted
+
+
+def get_error_details(error):
+    return list(get_formatted_error_traceback(error))
```

Comment:
Can't we use `traceback.format_exception(type(error), error, error.__traceback__)`?